src/pages/_nav_bar.tsx

```diff
@@ -16,7 +17,7 @@ function NavBarComponent({ customMouseEnter, customMouseLeave }: any) {
     "About Me",
     "Experience",
     "Projects",
-    "Contacts",
+    "Contact",
```

Comment:
The string should be 'Contacts' not 'Contact' since it consists of more than one contact address.